xml/admin_saltcluster.xml

```diff
@@ -124,10 +125,15 @@ placement:
     </para>
 <screen>
 service_type: rgw
+service_id: <replaceable>example_realm</replaceable>.<replaceable>example_zone</replaceable>
```

Comment:
Probably should be <replaceable>EXAMPLE_NFS</replaceable>

Reply:
To match: https://github.com/SUSE/doc-ses/pull/688/files#diff-9ac8700189c65d74f91272566728b3cb9bbe0f376c3e796bdd00e9accb4c2cbeR1765